developer/architecture/schemas.md

```diff
@@ -110,74 +109,135 @@ export const SchemaExample =  new SimpleSchema({
   },
   createdAt: {
     type: Date,
-    autoValue: function () {
-      if (this.isInsert) {
-        return new Date;
-      } else if (this.isUpsert) {
-        return {
-          $setOnInsert: new Date
-        };
-      }
-    }
+    autoValue: createdAtAutoValue
   }
 });
 ```
 
-#### Reaction.shopIdAutoValue
+#### createdAtAutoValue
```

Comment:
I can't find this method defined or used in core anywhere: https://github.com/reactioncommerce/reaction/blob/v1.8.0/lib/collections/schemas/helpers.js

Did it get lost?!